### ISSUE TITLE
Remove objects retroactively if permissions change

### ISSUE THREAD

**Issue by brianpeiris** [mozilla/hubs#1529]
If a user has created a camera/pen/media and then the relevant permission is denied, it'd be nice to have client side logic just remove all the relevant spawned objects since likely the intent of the room owner is to deny these things retroactively

_Originally posted by @gfodor in https://github.com/mozilla/hubs/pull/1517#issuecomment-509780164_

**Comment by brianpeiris** [comment]
Gonna punt on this for now. Feels a bit too destructive, IMO, at least with the current UX. Also trying to cut down on the scope of #1517

**Comment by misslivirose** [comment]
If we do decide to implement this in the future, I think it would be good to include a warning about it given the possible reach of how much it could delete. My mental model would be that it would only apply to future objects added, but I can see both sides of that.

**Comment by gfodor** [comment]
Yup this is pretty subjective, punting seems fine to me too. Basically this was to cover the scenario where a user discovers active defacement or abuse happening and flipping the permissions off is a response mechanism. (They obviously can still go remove the objects and kick people out manually)
